opentelemetry-sdk/src/opentelemetry/sdk/_metrics/point.py

```diff
@@ -12,16 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from dataclasses import dataclass
+import json
+from dataclasses import asdict, dataclass
+from enum import IntEnum
 from typing import Sequence, Union
 
+from opentelemetry.sdk.resources import Attributes, Resource
```

Comment:
Any reason using the this `Attributes` from `sdk.resources` instead of common definition?

Reply:
Nice catch, auto-import mistake :)